### PR TITLE
US-001: should now wait for a list of job roles in the GET route

### PR DESCRIPTION
**Pull request by liammc98**
We now load the Jobs List page and wait for the data to be sent over. 
This change applies to the front end only.

Code was manually tested.

### CODE REVIEW

File: routes.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 const express = require('express')
 const app = express();
 const router = express.Router()
+const bodyParser = require('body-parser')
 const nunjucks = require('nunjucks'); 
+const fetch = require('node-fetch')
+const { response } = require('express');
 //const employeeData = require('./employee.js');       // reference the file containing the function needed
```
Comment (jkelso662) [inline routes.js:8]: Would avoid adding commented lines into master!

File: routes.js [hunk 2]
```diff
@@ -12,11 +15,28 @@ nunjucks.configure('views', {
...
     res.render('jobslist'); 
+
+    console.log("Now trying to get the jobs list from the database");
+    
+    try{
+        const response = await fetch('http://localhost:7999/jobslist', {method: 'GET', headers:{} })
+        const data = await response.text();
+
+        console.log("Got the jobs list");
+        // res.setHeader("Content-Type", "text/html");
```
Comment (jkelso662) [inline routes.js:35]: More commented lines. Better avoid adding these to master.

**Comment by jkelso662** [comment]
Couple of commented lines in code maybe best not merged into your master branch. Also no description on the pull request. If these are resolved I am happy to accept merge request.

**Review by jkelso662 — APPROVED** [review]
Much better! If this is a completed chunk of work now ready for master then happy to approve!

**Review by RyanCunning1 — APPROVED** [review]

**Review by John-Neill-Kainos — APPROVED** [review]

**Pull request merged**